packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -205,6 +206,11 @@ class WebView extends StatefulWidget {
   /// [WebViewController.evaluateJavascript] can assume this.
   final PageFinishedCallback onPageFinished;
 
+  /// Allows remote WebView debugging if set to true
```

Comment:
This Dart doc doesn't add much information.

Some prompts:
  * What's remote webview debugging?
  * Is it supported on Android and iOS? does it work the same on both platforms?

Reply:
updated